import-export-cli/cmd/exportOperationPolicy.go

```diff
@@ -0,0 +1,103 @@
+
+	"github.com/wso2/product-apim-tooling/import-export-cli/credentials"
+
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+
+	"path/filepath"
+)
+
+var exportAPIPolicyName string
+var exportAPIPolicyVersion string
+var runningExportAPIPolicyCommand bool
+
+// ExportAPIPolicy command related usage info
+const ExportAPIPolicyCmdLiteral = "api"
+const exportAPIPolicyCmdShortDesc = "Export API Policies"
+const exportAPIPolicyCmdLongDesc = "Export API Policies from an environment"
+
+const exportAPIPolicyCmdExamples = utils.ProjectName + ` ` + ExportCmdLiteral + ` ` + ExportPolicyCmdLiteral + ` ` + ExportAPIPolicyCmdLiteral + ` -n addHeader -e dev
+ NOTE: All the 2 flags (--name (-n) and --environment (-e)) are mandatory.`
+
+// ExportAPIPolicyCmd represents the api policy export command
+var ExportAPIPolicyCmd = &cobra.Command{
+	Use: ExportAPIPolicyCmdLiteral + " (--name <name-of-the-api-policy> --environment " +
+		"<environment-from-which-the-api-policies-should-be-exported>)",
+	Short:   exportAPIPolicyCmdShortDesc,
+	Long:    exportAPIPolicyCmdLongDesc,
+	Example: exportAPIPolicyCmdExamples,
+	Run: func(cmd *cobra.Command, args []string) {
+		utils.Logln(utils.LogPrefixInfo + ExportAPIPolicyCmdLiteral + " called")
+		var apiPoliciesExportDirectory = filepath.Join(utils.ExportDirectory, utils.ExportedPoliciesDirName, utils.ExportedAPIPoliciesDirName)
+
+		cred, err := GetCredentials(CmdExportEnvironment)
+		if err != nil {
+			utils.HandleErrorAndExit("Error getting credentials", err)
+		}
+
+		exportAPIPolicyVersion = utils.APIPolicyVersion
+		executeExportAPIPolicyCmd(cred, apiPoliciesExportDirectory, exportAPIPolicyVersion, exportAPIPolicyName)
+	},
+}
+
+func executeExportAPIPolicyCmd(credential credentials.Credential, exportDirectory string, exportAPIPolicyVersion string, exportAPIPolicyName string) {
+	runningExportAPIPolicyCommand = true
+	accessToken, preCommandErr := credentials.GetOAuthAccessToken(credential, CmdExportEnvironment)
+	if preCommandErr == nil {
+		resp, err := impl.ExportAPIPolicyFromEnv(accessToken, CmdExportEnvironment, exportAPIPolicyName, exportAPIPolicyVersion)
+		if err != nil {
+			utils.HandleErrorAndExit("Error while exporting", err)
+		}
+		// Print info on response
+		utils.Logf(utils.LogPrefixInfo+"ResponseStatus: %v\n", resp.Status())
+		if resp.StatusCode() == http.StatusOK {
+			impl.WriteAPIPolicyToFile(exportDirectory, resp, exportAPIPolicyVersion, exportAPIPolicyName, runningExportAPIPolicyCommand)
+		} else if resp.StatusCode() == http.StatusInternalServerError {
+			// 500 Internal Server Error
+			fmt.Println(string(resp.Body()))
+		} else {
+			// neither 200 nor 500
+			fmt.Println("Error exporting API Policies:", resp.Status(), "\n", string(resp.Body()))
+		}
+	} else {
+		// error exporting Operarion Policy
+		fmt.Println("Error getting OAuth tokens while exporting API Policies:" + preCommandErr.Error())
```

Comment:
```suggestion
		fmt.Println("Error getting OAuth tokens while exporting the API Policy:" + preCommandErr.Error())
```
Shall we change all the places where the plural form is used, to singular?

Reply:
Fixed by [02a788a](https://github.com/wso2/product-apim-tooling/pull/924/commits/02a788a1570035b84d7acfea4bc999161a1bae47)